### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,82 @@
...
+* [Data Integrity v1.0](https://w3c.github.io/vc-data-integrity/)
+
+## Install
+
+Requires Node.js 16+
+
+To install via NPM:
+
+```
+npm install data-integrity-context
```
Comment (davidlehn) [inline README.md]:
```suggestion
npm install @digitalbazaar/data-integrity-context
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,82 @@
...
+```
+
+## Usage
+
+```js
+const dataIntegrity = require('data-integrity-context');
```
Comment (davidlehn) [inline README.md]:
```suggestion
const dataIntegrity = require('@digitalbazaar/data-integrity-context');
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,82 @@
-# data-integrity-context
-A Data Integrity context library for JavaScript.
+# Data Integrity Context Repository _(data-integrity-context)_
```
Comment (davidlehn) [inline README.md]: "Repository" doesn't seem needed.
```suggestion
# Data Integrity Context _(@digitalbazaar/data-integrity-context)_
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  for a key pair when generating a key. It may also be used to update
+  the public key ID after the key has been generated. The other term,
+  `publicAliasTemplate` serves a similar purpose, but instead holds
+  a URI template value (per RFC 6570) that can be used during key
+  generation to generate the public key ID from variables that appear
+  in the key's public description.
+
+### Changed
+- **BREAKING**: Published package under `@digitalbazaar/data-integrity-context`.
+
```
Comment (dlongley) [inline CHANGELOG.md]: This is a new package, this needs to be cleaned up.

File: contexts/data-integrity-v1.jsonld [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+              "@type": "@id",
+              "@container": "@set"
+            },
+            "keyAgreement": {
+              "@id": "https://w3id.org/security#keyAgreementMethod",
+              "@type": "@id",
+              "@container": "@set"
+            }
+          }
+        },
```
Comment (dlongley) [inline contexts/data-integrity-v1.jsonld:61]: I think this should work fine and be more future proof. Reminder: the build will have to be re-run to generate the other file.

```suggestion
        "proofPurpose": "https://w3id.org/security#proofPurpose",
```

**Review by dlongley — APPROVED** [review]
Thanks! Approving, but see comments.

File: contexts/data-integrity-v1.jsonld (hunk 5 above)
Comment (dlongley) [reply contexts/data-integrity-v1.jsonld:61]: On second thought, don't do this right now. We'll want to try to do it later though.

Comment (dlongley) [reply contexts/data-integrity-v1.jsonld:61]: Filed this: #2

**Pull request merged**